src/common/fs/FileUtils.cpp

```diff
@@ -530,6 +530,13 @@ void FileUtils::Iterator::openFileOrDirectory() {
     status_ = Status::OK();
 }
 
+int FileUtils::sync(int fd) {
+#ifdef HAVE_FDATASYNC
+    return fdatasync(fd);
```

Comment:
Be careful!  We should NOT use "fdatasync" here.

Reply:
ok, I will fix this.